src/test/java/dev/stalla/PodcastRssParserInteropTest.java

```diff
@@ -0,0 +1,332 @@
+package dev.stalla;
+
+import dev.stalla.model.Episode;
+import dev.stalla.model.Podcast;
+import dev.stalla.model.atom.AtomPerson;
+import dev.stalla.model.atom.Link;
+import dev.stalla.model.googleplay.GoogleplayCategory;
+import dev.stalla.model.itunes.ItunesCategory;
+import dev.stalla.model.podcastindex.Funding;
+import dev.stalla.model.podcastindex.Soundbite;
+import dev.stalla.model.podcastindex.Transcript;
+import dev.stalla.model.podlove.SimpleChapter;
+import dev.stalla.model.rss.RssCategory;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.w3c.dom.Document;
+import org.xml.sax.InputSource;
+import org.xml.sax.SAXException;
+
+import java.io.*;
```

Comment:
I would avoid wildcard imports, like we do in Kotlin. What do you think? I believe we simply haven't set up the Java code style (and, currently, we have no static analysis for Java anyway)

Reply:
I'm fine with skipping wildcard.